spec/models/form_spec.rb

```diff
@@ -175,20 +175,21 @@
 
   describe "#has_draft_version" do
     let(:live_form) { create(:made_live_form).form }
-    let(:new_form) { create(:form) }
+    let(:form) { create(:form) }
 
-    it "returns true if form has not been made live before" do
-      expect(new_form.has_draft_version).to eq(true)
+    it "returns true if form is draft" do
+      form.state = :draft
+      expect(form.has_draft_version).to eq(true)
     end
 
-    it "returns false if form has been made live and not edited" do
-      expect(live_form.has_draft_version).to eq(false)
+    it "returns true if form is live with a draft" do
+      form.state = :live_with_draft
+      expect(form.has_draft_version).to eq(true)
     end
 
-    it "returns true if form has been made live and been edited" do
-      live_form.update!(name: "Form (edited)")
-
-      expect(live_form.has_draft_version).to eq(true)
+    it "returns false if form is live and no edits" do
+      form.state = :live
+      expect(form.has_draft_version).to eq(false)
```

Comment:
Ideally we would keep the tests as unchanged as possible in the commits changing the method implementation, so we can be confident that the code changes result in the same behaviour.

Can we split the refactor of the tests into a separate commit?

Reply:
Done, thanks for the suggestion. Its easier to follow the changes now.